### DISCUSSION TITLE
API V3; createReceiptShipment; carrier_name; 'Asendia' tracking link

### DISCUSSION THREAD

**Discussion by MaxLBrick** [etsy/open-api#1025] [Q&A]
I use the 'createReceiptShipment' call to submit tracking information for a receipt.
This works fine in general but there is an issue with the 'Asendia' carrier.
The tracking link created for the 'Asendia' carrier isn't valid.

Tried passing 'globegistics' as carrier_name as described at
https://www.etsy.com/developers/documentation/getting_started/seller_tools#section_supported_carriers

Then found different list of supported carriers
https://developer.etsy.com/documentation/tutorials/fulfillment/#tracking-updates-for-shipping-carriers
I can see Globegistics in the list but the link leads to
https://a1.asendiausa.com/tracking/?trackingnumber={trackingnumber}
rather that to expected
https://tracking.asendia.com/tracking/{trackingnumber}

Is there a way to add an 'Asendia' carrier with a link to https://tracking.asendia.com/ or something else might be done to form a link to https://tracking.asendia.com/ ?

**Comment by Gareth064** [comment]
Hey bud 

This is gonna be a question for the Etsy Dev team to answer.

Drop them an email at developer@etsy.com.

They don't answer things here. This is more of a community discussion board now.

**Reply by MaxLBrick** [reply]
Thanks, @Gareth064 !
> Drop them an email at [developer@etsy.com](mailto:developer@etsy.com).

Did that too. Posted the question here looking if someone from the community had similar issue.

Got a reply from Etsy(if anyone would find it helpful):

> There is actually a way for you to update the tracking information when using that carrier as long as the correct "carrier_name" is passed along via API. We have two options for Asendia which are:
>  
> Asendia UK with the carrier_name: "asendia-uk" and;
> Asendia USA with the carrier_name: "asendia-usa". 
>  
> All of Etsy's supported carriers via API can be found here: https://developers.etsy.com/documentation/tutorials/fulfillment#tracking-updates-for-shipping-carriers.
